packages/webpack-cli/__tests__/ZeroConfigGroup.test.js

```diff
@@ -40,6 +41,8 @@ describe('GroupHelper', function () {
         ]);
 
         const result = group.run();
+        // ensure no other properties are added
+        expect(result.options).toMatchObject({ mode: 'development' });
```

Comment:
Can we look on plugins here?

Reply:
Old method is only adding config options and no plugins

Reply:
We should look what we don't have two terser plugins here

Reply:
Can you suggest ways to check it here? We're just passing mode to the compiler, so since core doesn't have two, we won't have two either.

Reply:
Hm, maybe we can add a test there manually added `Terser` to minimizer with custom options, for extractFilename, not the best solution, but will be work

Reply:
Added, something wrong with CI, will take a look tomm

Reply:
@evilebottnawi seems good now, PTAL